unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Font/DCLFont.cs

```diff
@@ -101,7 +117,7 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                 }
                 else
                 {
-                    Debug.Log($"couldn't fetch font from resources {resourceName}");
+                    Debug.Log($"couldn't fetch font from resources {fontResourceName}");
```

Comment:
Should we load the default font as a fallback?

Reply:
The "default" font is already set in the component prefab...
since TextShape has LiberationSans and UIText had San Francisco (now replaced for Inter) as default font I opted for a "couldn't load font then don't modify the already existing font for that component" approach.
but I could load a default font in that scenario if you prefer that